docs/src/lib/functions.md

```diff
@@ -80,3 +78,26 @@ transform!
 unique!
 vcat
 ```
+
+## Unsorted
+
+```@docs
+first
+last
+unique
+propertynames
+similar
+ourshow
+ourstrwidth
+sortperm
+pairs
+parent
+issorted
+getmaxwidths
+getprintedwidth
+gennames
+getchunkbounds
+showrowindices
+compacttype
+showrows
+```
```

Comment:
These are all the docstrings that were not included in the manual at the moment. These should be put into their appropriate sections in the manual before merging this, but I would prefer to leave this to someone who is more familiar with the overall structure of the package :slightly_smiling_face:

Reply:
Thank you for doing this. Are we allowed to exclude the unexported functions from this list? (e.g. `ourshow` or `gennames`)

Reply:
They can't be completely removed from the manual, without Documenter complaining, but I would propose having a (hidden) internals page to get around this (3885ff3ad14ff7be3269a64b41ebd584c753f3db).

Reply:
OK - thx.